sympy/integrals/rubi/constraint.py

```diff
@@ -0,0 +1,79 @@
+from sympy.external import import_module
+matchpy = import_module("matchpy")
+
+if matchpy:
+    Constraint, substitute = matchpy.Constraint, matchpy.substitute
+else:
+    raise ImportError('MatchPy could not be imported')
```

Comment:
This should be avoided. We need to find a way to skip this module without raising an error, otherwise the tests will fail.